### PR TITLE
DO-269: Rating component added.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/common/RatingBar/RatingBar.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { setRatingClass } from '../../../utils/common/formatters/formmatters';
+import { RATING_BAR_LEVEL_NAME } from '../../../constants/common/commonConstants';
+
+const RatingBar = (props) => {
+  const { progress } = props;
+  const ratingClass = setRatingClass(progress);
+  return (
+    <Container className="rating-bar__container">
+      <Container className="rating-bar__content">
+        <span className={ratingClass}>
```
Comment (mohitsaladiya) [inline src/components/common/RatingBar/RatingBar.jsx:14]: Can we rename the ratings to:

1. POOR
2. OKAY
3. GOOD
4. GREAT
5. EXCELLENT

Comment (amitjain24524) [reply src/components/common/RatingBar/RatingBar.jsx:14]: Rating name updated.

File: src/components/common/ProfileCompleteness/ProfileCompleteness.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Container from '../../common/Typography/Container/Container';
+import { getRatingName } from '../../../utils/common/formatters/formmatters';
+import './_ProfileCompleteness.scss';
+
+const ProfileCompleteness = (props) => {
```
Comment (mohitsaladiya) [inline src/components/common/ProfileCompleteness/ProfileCompleteness.jsx:7]: Please Rename it to ProfileStrength or whatever text Ajay has provided.

File: src/constants/common/commonConstants.js [hunk 3]
```diff
@@ -148,3 +148,35 @@ export const PROGRESS_BAR_LEVEL = {
...
+
+export const RATING_BAR_CLASS = {
+    TERRIBLE: 'terrible',
+    SOSO: 'soso',
+    OKAY: 'okay',
+    GOOD: 'good',
+    GREAT: 'great',
+  }
+
+export const RATING_BAR_LEVEL_NAME = {
```
Comment (mohitsaladiya) [inline src/constants/common/commonConstants.js:176]: Rename object keys as well

File: src/utils/common/formatters/formmatters.js [hunk 4]
```diff
@@ -113,3 +120,45 @@ export const setProgressBarTextColor = ((progress) => {
   }
   return textColor;
 });
+
+//Using for providing a class to Rating bar according to progress level
+export const setRatingClass = ((progress) => {
+  let ratingClass = '';
+  if (progress <= RATING_BAR_LEVEL.GREAT) {
```
Comment (mohitsaladiya) [inline src/utils/common/formatters/formmatters.js:127]: This constants will also get renamed.

File: src/components/common/ProfileCompleteness/ProfileCompleteness.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/ProfileCompleteness/ProfileCompleteness.jsx]: done

File: src/constants/common/commonConstants.js (hunk 3 above)
Comment (amitjain24524) [reply src/constants/common/commonConstants.js:176]: done

File: src/utils/common/formatters/formmatters.js (hunk 4 above)
Comment (amitjain24524) [reply src/utils/common/formatters/formmatters.js]: done

**Pull request merged**